webpack.config.js

```diff
@@ -189,32 +189,59 @@ if (!IS_PRODUCTION && REFRESH) {
   );
 }
 
+/**
+ * Webpack entry for password strength checker
+ */
+var pwConfig = {
```

Comment:
I think that this is kinda a breaking point for our webpack config to be become multiple files. 

maybe keep webpack.config.js where it is and add a webpack/ dir next to it with stuff like this we can import?